### ISSUE TITLE
Feature request: support horizontal tables

### ISSUE THREAD

**Issue by ramhoj** [jnicklas/capybara_table#1]
Where the `<th>`s are in a column, not the first row.

**Comment by jnicklas** [comment]
I think this one is even harder to express in XPath. Not even sure it's possible.

**Comment by ramhoj** [comment]
Yeah, we looked at the current xpath in complete silence for a while and than created this issue as some sort of "well at least we bothered someone else, that's something, right?". Feel free to close it unless you want it around in case someone with too much time comes around wants to give it a try.
